pkg/agent/util/net_windows.go

```diff
@@ -1097,3 +1209,91 @@ func renameHostInterface(oriName string, newName string) error {
 	_, err := runCommand(cmd)
 	return err
 }
+
+func parseAdapters(name string) ([]Adapter, error) {
```

Comment:
The name may be misleading. It sends request to the system API to get actual interfaces. It should be named something like `getAdaptersByName`.
And is it possible to get multiple adapters given a name? If not, it should be `getAdapterByName` with `(*Adapter, error)` returned

Reply:
Changed to `getAdaptersByName`. I checked the doc, and found that `Get-NetAdapter` supports wildcard characters like "E*2", I don't have enough knowledge on where we should use `getAdapterByName`. cc @wenyingd

Reply:
Discussed offline.